x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/utils.test.ts

```diff
@@ -223,3 +224,9 @@ describe('crawlDomainValidationToResult', () => {
     } as CrawlerDomainValidationStep);
   });
 });
+
+describe('getDeleteDomainSuccessMessage', () => {
+  it('includes the url', () => {
+    expect(getDeleteDomainSuccessMessage('https://elastic.co/')).toContain('https://elastic.co');
+  });
```

Comment:
Any reason not to test the full string (with equality)?

Reply:
I'm not a big fan of testing copy tbh.  Ideally writer like Chris should be able to go into our codebase and update copy without having to update unit tests.  Something like a button with `Loading...` I feel is less likely to have its copy updated.